src/Broadcasting/src/AuthorizationStatus.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Broadcasting;
+
+use Psr\Http\Message\ResponseInterface;
+
+final class AuthorizationStatus
+{
+    private bool $success;
+    private array $topics;
+    private array $attributes;
+    private ?ResponseInterface $response;
+
+    public function __construct(
+        bool $success,
+        array $topics,
+        array $attributes = [],
+        ?ResponseInterface $response = null
+    ) {
+        $this->success = $success;
+        $this->topics = $topics;
+        $this->attributes = $attributes;
+        $this->response = $response;
+    }
```

Comment:
Can be replaced with promouted properties;
Also getters can be replaced with readonly properties